### ISSUE TITLE
Can the plugin be built with different defaulted preset settings?

### ISSUE THREAD

**Issue by dlamblin** [uwolfer/gerrit-intellij-plugin#423]
It looks like settings will be stored in gerrit_settings.xml file.
https://github.com/uwolfer/gerrit-intellij-plugin/blob/7414bd4a401a38e21e510282014ba2189c7ed865/src/main/java/com/urswolfer/intellij/plugin/gerrit/GerritSettings.java#L41
However there is no default version of this file in the project, so I'm still browsing for where the defaults come from. Is it possible for a group to build their own version of this upstream plugin but bundle in a set of presets specific for that group?

**Comment by uwolfer** [comment]
You are right, gerrit_settings.xml is in the bundle, it gets created at runtime based on https://github.com/uwolfer/gerrit-intellij-plugin/blob/intellij2016.2/src/main/java/com/urswolfer/intellij/plugin/gerrit/GerritSettings.java.

I'm not sure how to share plugins settings, but you could look into official IntelliJ functionality: https://www.jetbrains.com/help/idea/sharing-your-ide-settings.html

Please post here if you have success doing this.
